app/views/layouts/application.html.erb

```diff
@@ -1,47 +1,42 @@
 <% content_for :body do %>
-  <% if use_govuk_account_layout? %>
-    <div id="wrapper">
-      <% content_for :before_content do %>
-        <%= yield :back_link %>
-      <% end %>
-      <%= yield :before_content %>
-      <main role="main" id="content">
-        <%= yield %>
-      </main>
-    </div>
-  <% else %>
-    <div class="govuk-width-container" id="wrapper">
-      <%= yield :back_link %>
-      <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
-        <div id="email-alert-frontend">
-          <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-              <%= yield %>
-            </div>
+  <div class="govuk-width-container" id="wrapper">
+    <%= yield :back_link %>
+    <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
+      <div id="email-alert-frontend">
```

Comment:
Is this div needed? I can't see any reference to it elsewhere in the code.

Reply:
Good question. It's in the existing code, but removing it seems to make no difference, and as you say there isn't any CSS or JS referring to it. I'll take it out.

Reply:
(Done)